doc/biocpp/presentation/index.html

```diff
@@ -735,29 +735,29 @@ <h3>Schedule<small>*</small>:</h3>
 <!-- .element: class="fragment" -->
 
 * <!-- .element: class="fragment" -->
-  start with "hello world" programm. <!-- hello world -->
+  Start with "hello world" program. <!-- hello world -->
 * <!-- .element: class="fragment" -->
-  read and use command-line arguments
+  Read and use command-line arguments.
 * <!-- .element: class="fragment" -->
-  use a library to read in a **fastq** file <!-- fastq -> fasta -->
+  Use a library to read in a **FASTQ** file. <!-- fastq -> fasta -->
 * <!-- .element: class="fragment" -->
-  use a library to write out a **fasta** file
+  Use a library to write out a **FASTA** file.
 </script>
 </section>
 
 <section>
 <p class=live-coding></p>
 
 <aside class="notes" data-markdown>
-    Start writing a simple program with hello world.
-    Learning goals:
-        * How to use Standard library functionality
-        * What does #include mean
-        * How to use standard library documentation: cppreference.com -> shared document.
-        * Entry point of any program -> the main
-        * String literal -> has to be the entry point of any program that captures command line arguments
+    Start writing a simple program that prints "hello world".
```

Comment:
Is this correct?

Reply:
`<aside class="notes" data-markdown>` are notes that only the speaker can see.

Reply:
Thanks, I Didn't notice.
But it would still be nice, if the speaker knows what they should do :D
(also for if this is reused at a later tutorial)

Reply:
Yeah sure, but I just wanted to explain why some parts are extremely cryptic :)